apps/guilds-subgraph/src/mappings/Create2Deployer/mapping.ts

```diff
@@ -44,19 +61,36 @@ export function handleDeployedEvent(event: Deployed): void {
     // At this point Guild wasn't initialized so we only create guild with default params
     guild.isActive = false;
     guild.proposals = [];
-    guild.members = [];
-    guild.bytecodeHash = event.params.bytecodeHash.toHexString();
-    guild.type = type;
+    guild.bytecodeHash = bytecodeHash;
+    guild.type = guildType;
 
     guild.save();
 
-    if (type == 'SnapshotRepERC20Guild') {
+    if (guildType == 'SnapshotRepERC20Guild') {
```

Comment:
Any reason we're using `==` and not `===`? `==` is almost always considered bad practice